baldrick/plugins/github_block_labels.py

```diff
@@ -0,0 +1,37 @@
+from loguru import logger
+
+from baldrick.plugins.github_pull_requests import pull_request_handler
+
+BLOCKED_MESSAGE = 'This pull request is blocked from being merged by labels.'
```

Comment:
could we have the name of the label, that triggered this, listed, too?

Reply:
Or all of them that triggers this in a repo? (I don't expect all maintainers to always know about this feature)